modules/steam/hm.nix

```
@@ -0,0 +1,60 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.stylix.targets.steam;
+
+  adwaitaCustomCss = config.lib.stylix.colors {
+    template = ./custom.mustache;
+    extension = "css";
+  };
+
+  adwaitaTheme = pkgs.stdenv.mkDerivation (_self: {
+    name = "Adwaita-for-Steam";
+    version = "3.1";
+
+    src = config.stylix.inputs.adwaita-for-steam;
+
+    buildInputs = with pkgs; [ python3 ];
+
+    buildPhase = ''
+      mkdir --parents $out/steamui/css
+      touch $out/steamui/css/library.css
+      cp ${adwaitaCustomCss} ./custom/custom.css
```

This might be neater:

```suggestion
      cp ${adwaitaCustomCss} custom/custom.css
```

---

I think that's up to personal preference. I'd personally continue to use `./` for consistency.